docs/FormatsAndConventions.md

```diff
@@ -138,6 +138,48 @@ e.g. `Error alert email has very long subject #5958`
 > to compensate for GitHub's lack of strong linking between the two.
 > Assume there is an invisible prefix in front of the PR title `Fixes issue : ...`
 
+## References to code elements
+
+Follow these conventions when referring to code elements **from a non-code context** e.g. when referring to a function 
+name from a commit message.
+
+> The objective is to be as concise as possible without causing ambiguities. Therefore, omit optional details when
+> those details are not pertinent to the context.
```

Comment:
How about "without `being ambiguous`"?
Then line [171](https://github.com/oss-generic/process/pull/57/files?#diff-0ec80c39f8532a34d5cc80d4123d4675R171) should read "as long as it `is not ambiguous`".